include/libdnf/module/module_errors.hpp

```diff
@@ -0,0 +1,82 @@
+/*
+Copyright Contributors to the libdnf project.
+
+This file is part of libdnf: https://github.com/rpm-software-management/libdnf/
+
+Libdnf is free software: you can redistribute it and/or modify
+it under the terms of the GNU Lesser General Public License as published by
+the Free Software Foundation, either version 2.1 of the License, or
+(at your option) any later version.
+
+Libdnf is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public License
+along with libdnf.  If not, see <https://www.gnu.org/licenses/>.
+*/
+
+#ifndef LIBDNF_MODULE_MODULE_ERRORS_HPP
+#define LIBDNF_MODULE_MODULE_ERRORS_HPP
+
+#include "libdnf/common/exception.hpp"
+
+
+namespace libdnf::module {
+
+
+class ModuleError : public Error {
+    using Error::Error;
+    const char * get_domain_name() const noexcept override { return "libdnf::module"; }
+    const char * get_name() const noexcept override { return "ModuleError"; }
+};
+
+
+// TODO(pkratoch): Naming of the module errors
+class ModuleNoModuleError : public ModuleError {
```

Comment:
What about `NoModuleError`?

Reply:
I renamed all the `ModuleError` subclasses.